test/Microsoft.AspNetCore.SignalR.Test.Server/wwwroot/lib/jasmine-2.5.2/boot.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
I'd rather not be checking these in but we would need gulp or something to copy them. I will try to do this separately.